types/url-regex-safe/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for url-regex-safe 1.0
+// Project: https://github.com/niftylettuce/url-regex-safe
+// Definitions by: Stuart Long <https://github.com/stuartlong>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Options = Partial<{
+    exact: boolean;
+    strict: boolean;
+    auth: boolean;
+    localhost: boolean;
+    parens: boolean;
+    apostrophes: boolean;
+    trailingPeriod: boolean;
+    ipv4: boolean;
+    ipv6: boolean;
+    tlds: string[];
+    returnString: boolean;
+}>;
+
+declare function createUrlRegExp(options?: Options): RegExp;
+
+export = createUrlRegExp;
```

Comment:
```suggestion
export as namespace urlRegexSafe;

export = createUrlRegExp;
```
I think it requires UMD support:
https://github.com/niftylettuce/url-regex-safe#vanillajs
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
https://unpkg.com/url-regex-safe@1.0.2/dist/url-regex-safe.min.js

Reply:
Sure thing, thanks!